uapolicy/securitypolicy.go

```diff
@@ -9,10 +9,10 @@ package uapolicy
 
 import (
 	"crypto/rsa"
-	"errors"
-	"fmt"
 	"sort"
 
+	"github.com/gopcua/opcua/errors"
+
```

Comment:
drop blank line

Reply:
are you using a different fmt tool? `go fmt` doesn't report any issues this or `uapolicy/crypto_hmac.go`

Reply:
No, I'm just a stickler for these things. I believe in consistency and that developers don't read code but pattern match it. So any deviation from the structure slows you down b/c you fall back to the slower "read code" mode.